### PR TITLE
Convert proof-of-concept code to functions

### LINKED ISSUE #3: Convert proof-of-concept code to test code
Extract the proof-of-concept code into Python functions or modules, as necessary.

### PR DESCRIPTION
**Pull request by gkhays**
This pull request wires up the new HTTP routes to the newly ported Raspberry Pi GPIO functions.

The original proof-off-concept code remains as tests under the test folder.

- Close #3

### CODE REVIEW

**Review by Zenguin15823 — COMMENTED** [review]
nice!

File: pod/ops.py
Comment (Zenguin15823) [inline pod/ops.py:1]: Make sure to do "GPIO.setmode(GPIO.BOARD)" to let the GPIO board know what numbering system you are using for the pins!

**Review by Zenguin15823 — COMMENTED** [review]
See comment on ops.py

File: pod/ops.py
Comment (gkhays) [reply pod/ops.py:1]: Already happening in setup method.

**Review by Zenguin15823 — COMMENTED** [review]
Good pair programming sesh!

**Review by Zenguin15823 — APPROVED** [review]
Approved!

**Pull request merged**